src/helpers/ratings.js

```diff
@@ -0,0 +1,29 @@
+import fetch from "node-fetch";
+const API_REVIEWS = "https://plankton-app-xhkom.ondigitalocean.app/api/reviews";
+const IMDB_API = "https://www.omdbapi.com/?apikey=81d6c4d0";
+
+export async function loadMovieRatings(id) {
+  const res = await fetch(`${API_REVIEWS}?filters[movie]=${id}&populate=movie`);
+  const payload = await res.json();
+  const imdbMovieId = payload.data[0].attributes.movie.data.attributes.imdbId;
```

Comment:
What happens if there are no reviews? In that case, I expect that `payload.data` is an empty array, that `payload.data[0]` is hence `undefined`, and that `payload.data[0].attributes` will throw an error.

Reply:
I verified this by adding a new movie to the CMS and visiting it's page, which crashed your server.

Always be careful when accessing an array like this. Whenever you see code like `myArray[0]`, that should raise a red flag. Only ever do something like that if you can first confirm that there is at least one item in the array, e.g. by checking `myArray.length`.